src/harness/unittests/session.ts

```diff
@@ -93,14 +93,14 @@ namespace ts.server {
 
                 session.executeCommand(req);
 
-                expect(lastSent).to.deep.equal(<protocol.Response>{
+                expect(lastSent).to.deep.equal(id<protocol.Response>({

```

Comment:
IMO, this is a failing of the chai type definition. I expect `expect` should capture an generic to expect as the argument for `equal`, which would allow you to cast the type under `expect` and receive proper completions in the `equal`. Failing that, this being a unit test, if we're removing `id`, a variable assignment is fine.